src/components/PeopleFilters.tsx

```diff
@@ -27,66 +78,46 @@ export const PeopleFilters = () => {
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
-            >
-              16
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=17"
-            >
-              17
-            </a>
 
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=18"
-            >
-              18
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=19"
-            >
-              19
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=20"
-            >
-              20
-            </a>
+            {centuriesValues.map(number => (
+              <Link
+                data-cy="century"
+                key={number}
+                className={classNames('button mr-1', {
+                  'is-info': centuries?.includes(String(number)),
+                })}
+                to={{
+                  search: getSearchWith({
+                    centuries: centuries.includes(String(number))
+                      ? centuries.filter(num => String(number) !== num)
+                      : [...centuries, String(number)],
+                  }, searchParams),
```

Comment:
move this logic to a variable.
Also you can simplify this logic since know you go through the array twice (includes and filter). So, you can just use filter or something